db/migrate/20220923185723_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- [ ] As per the given ERD, the reference name should be `author` instead of `user`, which should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...